sass/directives/_carousels.scss

```diff
@@ -105,7 +105,10 @@
   font-size: $direction-arrow-size / 2;
   cursor: pointer;
   border: 1px solid $inactive-carousel-button-color;
-  padding: $base-padding / 2;
+  text-align: center;
+  line-height: 28px;//this is for vertically centering arrows in ::after
+  height: 30px;
```

Comment:
Okay - then could you please convert these to `em`s then add a variable that represents the value and use that